src/Core/Multistore/MultistoreContextCheckerInterface.php

```diff
@@ -51,4 +51,9 @@ public function isGroupShopContext();
      * @return bool
      */
     public function isSingleShopContext();
+
+    /**
+     * @return int[]
+     */
+    public function getContextShopIds(): array;
```

Comment:
I believe, based on their naming, that this method would be better placed in `PrestaShop\PrestaShop\Core\Shop\ShopContextInterface` 
`MultistoreContextCheckerInterface` is more there to check if this or that is true, and `ShopContextInterface` about getting data related to the Shop context

Although if it was only me they're would probably deserve to be merged but that's not the point here